src/modules/invite/tests/invite.service.spec.ts

```diff
@@ -45,7 +45,7 @@ describe('InviteService', () => {
       time_left: 5,
       secret: 'secret',
       is_2fa_enabled: true,
-      testimonials: [],
+      products: []
```

Comment:
This is why your tests are failing.

Add the `testimonials: [],`

Reply:
Thanks for taking time out to review, I dont really understand this part of the review.

Reply:
DONE